node/impl/full/state.go

```diff
@@ -61,6 +63,7 @@ type StateModuleAPI interface {
 	StateMinerProvingDeadline(ctx context.Context, addr address.Address, tsk types.TipSetKey) (*dline.Info, error)
 	StateMinerPower(context.Context, address.Address, types.TipSetKey) (*api.MinerPower, error)
 	StateNetworkVersion(ctx context.Context, key types.TipSetKey) (network.Version, error)
+	StateActorCids(ctx context.Context, nv network.Version) (map[string]cid.Cid, error)
```

Comment:
Let's not have this in the StateModuleAPI -- it can just be on the StateAPI below.